lib/src/Pos/Util/UserSecret.hs

```diff
@@ -131,9 +136,11 @@ instance Buildable AccountData where
         pairsF = later $ fold . map (uncurry $ bprint ("("%build%", "%build%")"))
 
 data WalletData = WalletData
-    { _wdRootKey  :: !EncryptedSecretKey
-    , _wdName     :: !Text
-    , _wdAccounts :: !(Vector AccountData)
+    { _wdRootKey   :: !EncryptedSecretKey
+    , _wdName      :: !Text
+    , _wdLastIndex :: Word32
```

Comment:
I'd be consistent and make this field strict (here and above). Will aprove because this code will be thrown-away anyway.